src/main/assets/js/go-back.ts

```diff
@@ -1,7 +1,11 @@
 const backLink: HTMLElement | null = document.querySelector('.govuk-back-link');
 if (backLink) {
   backLink.onclick = function (e) {
-    e.preventDefault();
-    history.go(-1);
+    if (document.location.pathname === '/jurisdiction/interstitial') {
+      document.location.pathname = '/check-jurisdiction';
```

Comment:
You should be able to use/import the URLs from [src/main/steps/urls.ts](https://github.com/hmcts/nfdiv-frontend/blob/master/src/main/steps/urls.ts) and then do:

```suggestion
    if (document.location.pathname === JURISDICTION_INTERSTITIAL_URL) {
      document.location.pathname = CHECK_JURISDICTION;
```